models/repo/repo.go

```diff
@@ -158,6 +158,7 @@ type Repository struct {
 	CodeIndexerStatus               *RepoIndexerStatus `xorm:"-"`
 	StatsIndexerStatus              *RepoIndexerStatus `xorm:"-"`
 	IsFsckEnabled                   bool               `xorm:"NOT NULL DEFAULT true"`
+	IsCodeIndexingEnabled           bool               `xorm:"NOT NULL DEFAULT true"`
```

Comment:
This require a migration, it should be similar to [`models/migrations/v219.go`](https://github.com/go-gitea/gitea/blob/main/models/migrations/v219.go), copy paste this whole struct(fixup the imports) and run Sync2 on it.

Reply:
Will do!

May take a day or two until I have some cycles to spend on updating this PR, but I'll get there eventually.